File: mini-package-gui/src/app/models/create.offer.response.ts
import { TeamName } from "./team.name";

export class CreateOfferResponse {
    teamName: TeamName;
    price: number;
    inquireGuid: string;

    constructor(teamName: TeamName = TeamName.NONE, price: number = 0, inquireGuid: string = '') {
        this.teamName = teamName;
        this.price = price;
        this.inquireGuid = inquireGuid;
    }
}

File: mini-package-api/mini-package-api/Models/Responses/CreateOfferResponse.cs
using mini_package_api.Enums;

namespace mini_package_api.Models.Responses
{
    public class CreateOfferResponse
    {
        public Guid InquireGuid { get; set; }

        public decimal Price { get; set; }

        public TeamNameEnum TeamName { get; set; }  
    }
}
